modules/host/kernel/networking.config

```diff
@@ -0,0 +1,28 @@
+# Copyright 2022-2024 TII (SSRC) and the Ghaf contributors
+# SPDX-License-Identifier: CC-BY-SA-4.0
+CONFIG_INET=y
+CONFIG_PACKET=y
+CONFIG_BRIDGE=y
+CONFIG_NETDEVICES=y
+CONFIG_USB_USBNET=y
+CONFIG_NETFILTER=y
+CONFIG_NETFILTER_XTABLES=y
+CONFIG_NETFILTER_XT_TARGET_LOG=y
+CONFIG_NETFILTER_XT_MATCH_PKTTYPE=y
+CONFIG_NETFILTER_XT_MATCH_CONNTRACK=y
+CONFIG_NF_NAT=y
+CONFIG_NFT_TUNNEL=y
+CONFIG_NFT_COMPAT=y
+CONFIG_NF_TABLES=y
+CONFIG_NF_TABLES_INET=y
+CONFIG_NF_CONNTRACK=y
+CONFIG_TUN=y
+CONFIG_SYN_COOKIES=y
+CONFIG_NETWORK_SECMARK=y
+CONFIG_AUDIT=y
+CONFIG_SECURITY_SELINUX=y
```

Comment:
I think `SELINUX` should go to baseline or optionally to a new access control fragment. It's also related to network but it's not network. Or how do you see this?

Reply:
The reason not adding `SELINUX` in baseline is because of it's dependency on SECURITY_NETWORK, AUDIT, NET and INET all of them also need to be added in baseline as well. 
![image](https://github.com/tiiuae/ghaf/assets/147831196/acb8d8f5-03fc-4bf2-a953-d4b9b3ae28ae)

Please suggest further.

Reply:
This is ok. Thanks for clarification.